types/splitio.d.ts

```diff
@@ -911,7 +911,7 @@ declare namespace SplitIO {
    *
    * @typedef {string} SplitFilterType
    */
-  type SplitFilterType = 'byName' | 'byPrefix';
+  type SplitFilterType = 'bySet' | 'byName' | 'byPrefix';
```

Comment:
missing the sets property in SplitView type.
Check https://github.com/splitio/react-native-client/pull/71